pkgs/by-name/te/tenere/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  makeWrapper,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "tenere";
+  rev = "0f3181a";
+  version = "0.11.2-${rev}";
```

Comment:
Thanks for pointing that out.  Question, does this mean the package ends up in nixos-unstable rather than release-25.05 for example?

Reply:
> Question, does this mean the package ends up in nixos-unstable rather than release-25.05 for example?

Nope, this will be included all branches it's merged into -- which could be unstable (currently 25.05) or stable (24.11)